Logger/FileLogger.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string? _FilePath;
+        public string? FilePath
+        {
+            get => _FilePath!;
+
+            set
+            {
+                if (value == null) { throw new ArgumentNullException(nameof(value)); }
+
+                _FilePath = value;
+            }
+        }
+
+        public FileLogger(string? filePath) { FilePath = filePath; }  
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            string logResult = $"{DateTime.Now} {ClassName} {logLevel}: {message} \n";
```

Comment:
String interpolation! The only suggestion I have here is to use `Environment.NewLine` instead of `\n`, but looks good otherwise. 
```suggestion
            string logResult = $"{DateTime.Now} {ClassName} {logLevel}: {message}{Environment.NewLine}";
```